de-identification-app/chart/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+image:
+  repository: alvearie/deid
+  pullPolicy: IfNotPresent
+  tag: 0.0.1
+service:
+  type: ClusterIP
+  port: 8080
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+imagePullSecrets: []
```

Comment:
Does a user need to fill a secret or leaving the value blank works if the image is not in the local repo ? Asking because it's not in the instructions if it's needed.

Reply:
The user does not need to fill a secret currently, but the template includes the place for that if the Dockerhub image changes to require it. This is part of a template that you get from helm when creating new charts.

Reply:
No changes required then. Thanks